core/commands/swarm.go

```diff
@@ -50,7 +52,14 @@ var swarmPeersCmd = &cmds.Command{
 'ipfs swarm peers' lists the set of peers this node is connected to.
 `,
 	},
+	Options: []cmds.Option{
+		cmds.BoolOption("verbose", "v",
+			`Also display latency along with peer information in the following form:
+        <peer address> <latency>
+        `),
```

Comment:
Please add `.Default(false)` it might not change much in this context but clarifies some things.